messaging-experiments.yaml

```diff
@@ -165,3 +165,70 @@
                     string_id: onboarding-browse-privately-button
                   action:
                     type: OPEN_PRIVATE_BROWSER_WINDOW
+- id: MULTISTAGE_ABOUT_WELCOME_TEST
+  enabled: true
+  filter_expression: env.version >= '78.' && env.version < '79.' &&
```

Comment:
Do we still need the `filter_expression` now? I think this field is only required for 77, can you confirm?

Reply:
In 76 we had limitation of using filter_expression only. We are using filter_expression and targeting both 77 onwards,  filter_expression gives flexibility to use  Remote Settings filter where as targeting is limited to ASRoutertargeting

Reply:
okay. I find this a bit confusing, for example, the Firefox version targeting was specified in both `filter_expression` and `targeting`, and they are not different. The one in `targeting` seems a bit redundant and inconsistent to me.